icicle_v3/backend/cpu/include/ntt_template.h

```diff
@@ -0,0 +1,264 @@
+#include "icicle/ntt.h"
+#include "icicle/errors.h"
+#include "icicle/runtime.h"
+#include "icicle/utils/log.h"
+
+#include "icicle/fields/field_config.h"
+#include "icicle/utils/log.h"
+
+#include <vector>
+#include <algorithm>
+#include <iostream>
+#include <cmath>
+#include <cstdint>
+
+using namespace field_config;
+using namespace icicle;
+namespace ntt_template {
+  template <typename S>
+  class CpuNttDomain
+  {
+    // TODO - coset, mixed-radix NTT
```

Comment:
please put your name on TODOs (e.g. TODO Shanie) so that it is clear who is going to do it.

Reply:
From now on I'll do that. Anyway, I'm removing this line, coset implementetion will be added in the next commit, and mixed radix is not for now (right?)